server/evaluator.py

```
@@ -0,0 +1,87 @@
+                if data.get("type") == "metrics" and data.get("payload", {}).get("type") == "turn_complete":
+                     self._evaluate_turn()
+                     # Reset for next turn
+                     self._user_finished_speaking_time = None
+                     self._last_ttfb = None
+                     self._user_utterance = ""
+                     self._bot_utterance = ""
+
+        await self.push_frame(frame, direction)
+
+    def _log_ttfb(self, ttfb: float):
+        logger.info(f"====== EVALUATION METRIC ======")
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)